specs/rider_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'csv'
+require_relative 'spec_helper'
+
+describe "rider class" do
+
+  it "requires three parameters for an instance of the rider class" do
+    rider_id = 540
+    name = "Bugs Bunny"
+    phone_num = "206-230-9140"
+
+    new_rider = RideShare::Rider.new(rider_id, name, phone_num)
+
+    new_rider.must_respond_to :rider_id
+    new_rider.rider_id.must_equal rider_id
+
+    new_rider.must_respond_to :name
+    new_rider.name.must_equal name
+
+    new_rider.must_respond_to :phone_num
+    new_rider.phone_num.must_equal phone_num
+  end
+
+  #for the self.all_rider_info method
+  it "returns an array or all rider info" do
+    test_class = RideShare::Rider.all_rider_info
+    test_class.must_be_instance_of Array
```

Comment:
I would like to see some coverage here about what's in the array. For example:
- Do you get the number of elements you expect?
- Are all elements instances of `Trip`?
- Did you get the first and last trips out of the CSV?